README.md

```diff
@@ -54,9 +54,7 @@ Reservoir Market v2 is lightly configurable with the configurations below. You c
 | Environment Variable | Required | Description | Example |
 |--------------------------------|----------|-------------------------------------------------------------------------------------|---------------------|
 | NEXT_PUBLIC_HOST_URL | `true` | The domain that the deployed project is hosted on. | http://localhost:3000 |
-| ETH_RESERVOIR_API_KEY | `false` | Ethereum Reservoir API key provided by the Reservoir Protocol. [Get your own API key](https://reservoir.tools/request-api-key). | 123e4567-e89b-12d3-a456-426614174000 |
-| GOERLI_RESERVOIR_API_KEY | `false` | Goerli Reservoir API key provided by the Reservoir Protocol. [Get your own API key](https://reservoir.tools/request-api-key). | 123e4567-e89b-12d3-a456-426614174000 |
-| POLYGON_RESERVOIR_API_KEY | `false` | Polygon Reservoir API key provided by the Reservoir Protocol. [Get your own API key](https://reservoir.tools/request-api-key). | 123e4567-e89b-12d3-a456-426614174000 |
+| ETH_RESERVOIR_API_KEY | `false` | Reservoir API key provided by the Reservoir Protocol. [Get your own API key](https://reservoir.tools/request-api-key). | 123e4567-e89b-12d3-a456-426614174000 |
```

Comment:
Can we just change this to RESERVOIR_API_KEY

Reply:
Also we'll need to make this change on the testnets branch too

Reply:
Can we also update it in our readme docs? https://docs.reservoir.tools/reference/open-source-marketplace